grid.css

```diff
@@ -13,6 +13,8 @@ heading1{
     border-bottom: 1px solid #aaa;
 	grid-template-rows: 50px;
 	background-color: white;
+	position: sticky;
```

Comment:
#made query headings for both playlist queries and user list query sticky when using the scroll area so they stay visible and at the top of the div